tests/dashbio_demos/app_onco_print.py

```diff
@@ -102,219 +102,206 @@ def header_colors():
-                    data=dataset3
-                ),
-            ], className='oncoprint-card eight columns'),
-            html.Div([
-                dcc.Tabs(
-                    id='oncoprint-tabs',
-                    value='oncoprint-tab-select',
-                    children=[
-                        dcc.Tab(
-                            label='Select',
-                            value='oncoprint-tab-select',
-                            children=[
-                                html.Div([
-                                    html.H4(
-                                        "Select Dataset"
-                                    ),
-                                    dcc.Dropdown(
-                                        id='oncoprint-dropdown',
-                                        className='oncoprint-select',
-                                        options=[
-                                            {
-                                                'label': '{}.json'.format(ds),
-                                                'value': ds
-                                            }
-                                            for ds in DATASETS
-                                        ],
-                                        value='dataset3',
-                                    ),
-                                ], className='oncoprint-subcard'),
-                                html.Div([
-                                    html.H4(
-                                        "Hover/Click/Event Data"
-                                    ),
-                                    dcc.Textarea(
-                                        id="oncoprint-events",
-                                        placeholder="Hover or click on data to see it here.",
-                                        value="Hover or click on data to see it here.",
-                                        className="oncoprint-events",
+
+        dash_bio.OncoPrint(
+            id='oncoprint-chart',
+            height=550,
+            data=[]
+        ),
+
+        html.Div(id='oncoprint-control-tabs', children=[
+            dcc.Tabs(
+                id='oncoprint-tabs',
+                children=[
+                    dcc.Tab(
+                        label='About',
+                        value='what-is',
+                        children=html.Div(className='oncoprint-tab', children=[
+                            html.H4(
+                                "What is OncoPrint?"
+                            ),
+                            html.P(
+                                """
+                                The OncoPrint component is used to view multiple genetic
+                                alteration events through an interactive and zoomable
+                                heatmap. It is a React/Dash port of the popular
+                                oncoPrint() function from the BioConductor R
```

Comment:
```suggestion
                                oncoPrint() function from the Bioconductor R
```
according to https://www.bioconductor.org/ (and, apparently, Bioconductor is a *collection* of R packages, the very package providing oncoPrint() being ComplexHeatmap)